contracts/utils/DAOFactory.sol

```diff
@@ -193,42 +218,52 @@ contract DAOFactory is Initializable {
     /**
      * @dev Set initial schemes for the organization.
      * @param _avatar organization avatar (returns from forgeOrg)
-     * @param _schemesNames the schemes name to register for the organization
-     * @param _schemesData the schemes initilization data
-     * @param _schemesInitilizeDataLens the schemes initilization data lens (at _schemesData)
-     * @param _permissions the schemes permissions.
-     * @param _metaData dao meta data hash
+     * @param _encodedSetSchemesParams _setSchemes parameters
+     * @param _packageVersion package version
      */
     function _setSchemes (
```

Comment:
Each scheme receives DAOFactory, and package version in its init params, which creates a large duplication of data. It might be too complicated to do but it would be better if we could instead of passing it to each scheme we could pass it here for the schemes